components/Layout.tsx

```diff
@@ -83,20 +92,38 @@ export default function Layout(props: LayoutProps) {
         href: "/signin",
         inner: "Sign in",
       },
-    {
-      href: "/submit",
-      inner: <span class={BUTTON_STYLES}>Submit</span>,
-    },
   ];
 
+  if (props.session) {
+    headerNavItems.push({
+      href: "/account/notifications",
+      inner: (
+        <span>
+          {props.hasNotifications!
+            ? (
+              <>
+                Notifications <span class="text-blue-500">•</span>
+              </>
+            )
+            : "Notifications"}
+        </span>
+      ),
+    });
```

Comment:
To clarify, I think:
1. The notifications link should be an icon.
2. The link should always be visible, with the blue dot only showing when unread notifications exist.
3. The link should still be visible if the user is signed out. The user would be automatically redirected to `/account/notifications` anyway, thanks to current middleware.